### PR TITLE
docs: move tech stack to a separate page

### PR DESCRIPTION
**Pull request by tianzhou**
The main page has too much content and we need to reserve some space for introducing tenant mode

### CODE REVIEW

File: pages/techstack.vue [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+          />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "@nuxtjs/composition-api";
+import EmojiSection from "~/components/EmojiSection.vue";
```
Comment (boojack) [inline pages/techstack.vue:507]: The `EmojiSection` component is not used.

Comment (tianzhou) [reply pages/techstack.vue]: Done

**Comment by tianzhou** [comment]
PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Review by ecmadao — APPROVED** [review]
LGTM

**Pull request merged**